cardano-node/src/Cardano/Tracing/OrphanInstances/Network.hs

```diff
@@ -924,12 +970,13 @@ instance ToObject (WithDomainName (SubscriptionTrace SockAddr)) where
              , "event" .= show ev ]
 
 
-instance (Show peer) => ToObject (WithMuxBearer peer MuxTrace) where
-  toObject _verb (WithMuxBearer b ev) =
+instance (ToObject peer) => ToObject (WithMuxBearer peer MuxTrace) where
```

Comment:
Redundant brackets, this may cause CI to fail

Reply:
done